src/neo/Trie/MPT/MPTNodeType.cs

```diff
@@ -0,0 +1,11 @@
+
+namespace Neo.Trie.MPT
+{
+    public enum NodeType
```

Comment:
There is no `NullNode`?

Reply:
I use empty `HashNode` as `NullNode`.
https://github.com/KickSeason/neo/blob/826fc16c62cdefc1dd771932a46705aac5a8d211/src/neo/Trie/MPT/MPTNode/HashNode.cs#L27-L32